server.js

```diff
@@ -1,5 +1,21 @@
-const app = require('./app')
+const mongoose = require('mongoose');
+const app = require('./app');
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+// const DB_HOST = process.env;
+const DB_HOST =
+  'mongodb+srv://vaniutasa:G74SZWhzalv6wXhe@cluster0.ft08ttn.mongodb.net/db-contacts?retryWrites=true&w=majority';
```

Comment:
connection string should be stored in .env file. This file should be added to gitignore and not be pushed to gh